tests/Unit/ControlSystem/Test_Tags.cpp

```diff
@@ -12,27 +12,43 @@
 #include <vector>
 
 #include "ControlSystem/Averager.hpp"
+#include "ControlSystem/Component.hpp"
 #include "ControlSystem/Controller.hpp"
 #include "ControlSystem/Tags.hpp"
 #include "ControlSystem/TimescaleTuner.hpp"
 #include "DataStructures/DataVector.hpp"
+#include "Domain/Creators/DomainCreator.hpp"
 #include "Domain/Domain.hpp"
 #include "Domain/FunctionsOfTime/FunctionOfTime.hpp"
 #include "Domain/FunctionsOfTime/PiecewisePolynomial.hpp"
 #include "Domain/OptionTags.hpp"
 #include "Framework/TestCreation.hpp"
 #include "Helpers/ControlSystem/TestStructs.hpp"
 #include "Helpers/DataStructures/DataBox/TestHelpers.hpp"
+#include "Time/Tags.hpp"
 #include "Utilities/ErrorHandling/Error.hpp"
+#include "Utilities/GetOutput.hpp"
 #include "Utilities/Literals.hpp"
 #include "Utilities/TMPL.hpp"
 #include "Utilities/TaggedTuple.hpp"
 
 namespace {
-const double initial_time = 2.0;
+const double initial_time = 0.0;
```

Comment:
How hard is it to keep this a nonzero value?  Assuming an initial time of zero somewhere would be a really easy error to make.

Reply:
Not hard. I changed it to keep the initial time step small in the test of the FunctionsOfTimeInitialize tag, but for testing it just has to be small enough to not interfere with the initial expiration times.. I'll revert this to 2.0 and change other things to accommodate.